src/containers/WelcomePage/components/Revisions.tsx

```diff
@@ -197,14 +192,7 @@ const Revisions = ({ userData }: Props) => {
             id: `title_${resource.id}`,
             data: (
               <CellWrapper>
-                <StyledTimeIcon
-                  size="small"
-                  data-status={warnStatus}
-                  title={revisions}
-                  aria-label={revisions}
-                  aria-hidden={warnStatus === "warn"}
-                  visibility={warnStatus === "warn" ? "hidden" : "visible"}
-                />
+                <StatusTimeFill variant={warnStatus} size="small" title={revisions} aria-label={revisions} />
```

Comment:
Driver vi ikke med `visibility` lenger?

Reply:
Nei, vi ble enige om på møtet at det skulle være samme oppførel for klokken overalt!